src/container/chart_template.ts

```diff
@@ -143,6 +144,20 @@ export class ChartTemplate {
           );
         }
 
+        // Guide
+        if (Prototypes.isType(item.chartElement.classID, "guide.guide")) {
+          const valueProp = this.template.properties.filter(
```

Comment:
Can filter return 0 items?

Reply:
Shouldn't be, but I'll add a guard for it